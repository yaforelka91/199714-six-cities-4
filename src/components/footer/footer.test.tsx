import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Footer from './footer';
import {Router} from 'react-router-dom';
import history from '../../history';

describe(`FooterSnapshot`, () => {
  it(`should render Footer`, () => {
    const tree = renderer.create(
        <Router history={history}>
          <Footer/>
        </Router>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
