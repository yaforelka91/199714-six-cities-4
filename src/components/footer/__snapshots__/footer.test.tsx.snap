// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterSnapshot should render Footer 1`] = `
<footer
  className="footer"
>
  <a
    className="footer__logo-link"
    href="/"
    onClick={[Function]}
  >
    <img
      alt="6 cities logo"
      className="footer__logo"
      height="33"
      src="/img/logo.svg"
      width="64"
    />
  </a>
</footer>
`;
